content/doc/book/system-administration/authenticating-scripted-clients.adoc

```diff
@@ -0,0 +1,181 @@
+---
+layout: section
+---
+
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../resources]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Authenticating scripted clients
+
+To make scripted clients (such as wget) invoke operations that require
+authorization (such as scheduling a build), use HTTP BASIC
+authentication to specify the user name and the API token. 
+
+Earlier versions of Jenkins require you to specify your real password,
+and it is only available when your security realm is password-based (for
+example, OpenID, Crowd and CAS plugins authenticate you without a
+password, so you simply don't have any password!) Specifying the real
+password is still supported, but it is not recommended
+because the risk of revealing password, and the human tendency to reuse
+the same password in different places.
+
+The API token is available in your personal configuration page. 
+Click your name on the top right corner on every page, 
+then click "Configure" to see your API token. 
+(The URL `+$root/me/configure+` is a good shortcut.) 
+You can also change your API token from here.
+
+Note that Jenkins does not do any authorization negotiation. 
+i.e. it immediately returns a 403 (Forbidden) response instead
+of a 401 (Unauthorized) response, so make sure to send the authentication
+information from the first request (aka "preemptive authentication").
+
+== Shell with curl
+
+The `curl` command is available for most operating systems including Linux, macOS, Windows, FreeBSD, and more.
+
+[source,sh]
+----
+curl -X POST -L --user jenkins:apiToken \
+    https://jenkins.yourcompany.com/job/your_job/build
+----
+
+== Shell with wget
+
+NOTE: The `wget` command needs the `--auth-no-challenge` option
+to authenticate to Jenkins:
+
+[source,sh]
+----
+wget --auth-no-challenge \
```

Comment:
tested this, and auth-no-challenge is still needed 👍

Reply:
Agreed, and left in the document.  Thanks!